### ISSUE TITLE
`oct` can't find it's own configuration(?)

### ISSUE THREAD

**Issue by nhosoi** [openshift/origin-ci-tool#124]
Hi @stevekuznetsov 
`oct` command fails crashes as follows.  This is the traceback from `oct deprovision`.  

```
 Traceback (most recent call last):
   File "/home/nhosoi/venv/bin/oct", line 11, in <module>
     load_entry_point('origin-ci-tool==0.1.0', 'console_scripts', 'oct')()
   File "/home/nhosoi/venv/lib/python2.7/site-packages/click/core.py", line 722, in __call__
     return self.main(*args, **kwargs)
   File "/home/nhosoi/venv/lib/python2.7/site-packages/click/core.py", line 697, in main
     rv = self.invoke(ctx)
   File "/home/nhosoi/venv/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
     return _process_result(sub_ctx.command.invoke(sub_ctx))
   File "/home/nhosoi/venv/lib/python2.7/site-packages/click/core.py", line 895, in invoke
     return ctx.invoke(self.callback, **ctx.params)
   File "/home/nhosoi/venv/lib/python2.7/site-packages/click/core.py", line 535, in invoke
     return callback(*args, **kwargs)
   File "/home/nhosoi/venv/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
     return f(get_current_context(), *args, **kwargs)
   File "/home/nhosoi/venv/lib/python2.7/site-packages/oct/cli/deprovision.py", line 36, in deprovision
     playbook_variables={'origin_ci_inventory_dir': context.obj.ansible_client_configuration.host_list, },
 AttributeError: 'NoneType' object has no attribute 'host_list'
```

Other `oct` commands such as `bootstrap self` and `configure ansible-client --view` fail in the similar manner except this command line:

```
$ oct configure ansible-defaults --view
         connection: ssh
      become_method: sudo
               user: origin
docker_volume_group: docker
             become: True
        become_user: origin
              hosts: OSEv3
```

I'm attaching config_origin-ci-tool.tar.zip which contains:

```
 .config/origin-ci-tool/ansible_client_configuration.yml
 .config/origin-ci-tool/ansible_variables.yml
 .config/origin-ci-tool/aws_client_configuration.yml
 .config/origin-ci-tool/aws_variables.yml
```

[config_origin-ci-tool.zip](https://github.com/openshift/origin-ci-tool/files/1123455/config_origin-ci-tool.zip)

Thanks!

**Comment by stevekuznetsov** [comment]
It does seem like the configuration got corrupted:

```
$ cat .config/origin-ci-tool/ansible_client_configuration.yml
--- null
...
```

Can you try to remove this file (`rm ~/.config/origin-ci-tool/ansible_client_configuration.yml`) and re-run `oct configure ansible-client --view` ?

**Comment by stevekuznetsov** [comment]
Also, do you remember what you were doing when this happened? I can't tell if this was due to the file writing being interrupted or somehow the data itself got corrupted and a successful write persisted bad data.

**Comment by lukas-vlcek** [comment]
FYI @stevekuznetsov @nhosoi I was running into similar issue and deleting the `~/.config/origin-ci-tool/ansible_client_configuration.yml` file solved it for me.

I am not 100% sure what caused this situation but I am sure it started happening after I interrupted execution of this script <https://github.com/richm/scripts/blob/bbcbbfed3749e72e4beb15d788a24ccc1910d627/oshift-oct-new-dev.sh> which I use to deploy logging during development. This script was running very slow one day so I terminated it manually and from that point I was hitting all kinds of issues all looking very similar to what Noriko reported here.
